addons/edi_sale/models/edi_sale_line_request_record.py

```diff
@@ -88,30 +88,72 @@ def execute(self):
         for r, batch in self.batched(self.BATCH_CREATE):
 
             _logger.info(
-                "%s creating %s %d-%d of %d", doc.name, SaleLine._name, r[0], r[-1], len(self)
+                "%s creating/updating %s %d-%d of %d", doc.name, SaleLine._name, r[0], r[-1], len(self)
             )
 
-            # Create order lines
+            # Create or update order lines
```

Comment:
I'm modifying the `execute` method to handle update (and delete) cases; yet to check if synchronizer can handle it. Any pointers on how to check/do that is welcome!